std-bits/table/src/main/java/org/enso/table/excel/ExcelFileFormat.java

```diff
@@ -2,5 +2,6 @@
 
 public enum ExcelFileFormat {
   XLS,
-  XLSX
+  XLSX,
+  XLSX_FALLBACK
```

Comment:
Is `XLSX_FALLBACK` used anywhere? Or is it here so that we could enable it in the future?

Reply:
for later: Perhaps we could have some tests that load tables from Excel files using the new and fallback mechanisms and compare them for consistency?

Reply:
Exactly why it is there - haven't added to the Enso side yet.